api/admin.go

```diff
@@ -145,6 +146,24 @@ func reloadConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
+func invalidateAllCaches(c *Context, w http.ResponseWriter, r *http.Request) {
+	debug.FreeOSMemory()
```

Comment:
Did you intend to leave this in here? If so, it should probably be done _after_ the caches have been invalidated